internal/statestore/backfill.go

```diff
@@ -26,7 +27,13 @@ import (
 	"open-match.dev/open-match/pkg/pb"
 )
 
-// CreateBackfill creates a new Backfill in the state storage if one doesn't exist. The xids algorithm used to create the ids ensures that they are unique with no system wide synchronization. Calling clients are forbidden from choosing an id during create. So no conflicts will occur.
+const (
+	allBackfills = "allBackfills"
+)
+
+// CreateBackfill creates a new Backfill in the state storage if one doesn't exist.
```

Comment:
What's with all the comment formatting changes?  They aren't relevant to this PR...

Reply:
Those are just some minor upgrades i found when reading the code that could be done without affecting the context of the PR

Reply:
Please avoid unrelated style cleanups in PRs.  Standalone changes to do cleanup is fine, as long as there's no fighting about style (which is just a waste of everyone's time.)